### ISSUE TITLE
add dependencies between services

### ISSUE THREAD

**Issue by chemidy** [yext/edward#117]
Sometimes we need to start microservices in a specific order.

It will be very nice to have a dependency management in the config file, and check before starting service3 if service 1 & service 2 are running else start service 1 & service 2.

` 
 {
            "name": "service3",
            "path": "auth/cmd/service3",
            "commands": {
                "build": "go install",
                "launch": "service3"
            },
            "dependency": ["service1","service2"]
        },
`
